src/org/traccar/protocol/Gps103ProtocolDecoder.java

```diff
@@ -189,7 +189,8 @@ protected Object decode(
             getLastLocation(position, dateBuilder.getDate());
 
             position.set(Position.KEY_ODOMETER, parser.nextInt());
-            position.set(Position.KEY_FUEL, parser.next());
+            parser.next();                                                          // instant fuel consumption
```

Comment:
Just one space is the standard. Only exception is regular expressions because there is a comment for each line.